server/routes/schools.js

```diff
@@ -0,0 +1,67 @@
+var express = require('express');
+var router = express.Router();
+var bodyParser = require("body-parser");
+
+const { PrismaClient } = require('@prisma/client')
+
+router.use(bodyParser.json());
+router.use(bodyParser.urlencoded({ extended: false }));
+
+async function getSchools(){
+  const allMySchools = await prisma2.school.findMany();
+  return allMySchools
+}
+
+async function postSchools(body){
+  let name = body.name;
+  let address = body.address;
+  let numberOfStudents = body.numberOfStudents;
+  let specialityPrograms = body.specialityPrograms;
+
+  if(!specialityPrograms || !name || !address || !numberOfStudents ){
+    return 422;
+  }
+ //make sure you have all the parameters
+  if(numberOfStudents < 0){
+   
+    return 422;
+  }
+
+  let test = await prisma2.school.create({data:{
+      name: body.name,
+      address:body.address,
+      numberOfStudents: body.numberOfStudents,
+      specialityPrograms: body.specialityPrograms
+  }})
+  return test;
+}
+
+async function deleteSchools(body){
+  let response = await prisma2.school.delete({
+    where:{
+      id: body.id
+    }
+  })
+
+  return response
+}
+
+const prisma2 = new PrismaClient()
+/* GET users listing. */
+router.get('/',  async function(req,res,next){
+  res.send(getSchools());
+});
+
+router.post('/', async function(req,res,next){
+  res.send(postSchools(req.body));
+})
+
+
+router.delete('/', async function(req,res,next){
+  res.send(deleteSchools(res.body));
+})
+
+module.exports = router;
+module.exports.deleteSchools = deleteSchools;
+module.exports.postSchools = postSchools;
```

Comment:
smart to export these so they are unit testable 👍 in next sprint we will probably be able to simulate sending a request directly through the express router instead, but for now this is great